Esignet/scripts/Esignet_Test_Script.jmx

```diff
@@ -9402,6 +9467,735 @@ log.info(&quot;Password: &quot; + password)
           <hashTree/>
         </hashTree>
       </hashTree>
+      <ThreadGroup guiclass="ThreadGroupGui" testclass="ThreadGroup" testname="Sign Up Service - Registration Status Complete Flow (Execution)" enabled="true">
+        <stringProp name="ThreadGroup.on_sample_error">continue</stringProp>
+        <elementProp name="ThreadGroup.main_controller" elementType="LoopController" guiclass="LoopControlPanel" testclass="LoopController" testname="Loop Controller" enabled="true">
+          <boolProp name="LoopController.continue_forever">false</boolProp>
+          <stringProp name="LoopController.loops">1</stringProp>
```

Comment:
Sign Up Service - Reset Password (Execution) - this should have a preparation for creating the users.

Reply:
Is this complete flow for reset password ?

Reply:
Yes this is the complete flow, but removing the Sign Up Service - Reset Password (Execution) thread group from this PR, as Nihal has worked on Reset password, and he has already raised another PR for that.